framework/generated/khronos_generators/vulkan_generators/decode_pnext_struct_generator.py

```diff
@@ -198,37 +216,8 @@ def endFile(self):
         # Finish processing in superclass
         BaseGenerator.endFile(self)
 
-    def genStruct(self, typeinfo, typename, alias):
-        """Method override."""
-        if not alias:
-            # Only process struct types that specify a 'structextends' tag, which indicates the struct can be used in a pNext chain.
-            parent_structs = typeinfo.elem.get('structextends')
-            if parent_structs:
-                stype = self.make_structure_type_enum(typeinfo, typename)
-                if stype:
-                    self.stype_values[typename] = stype
-
     def need_feature_generation(self):
         """Indicates that the current feature has C++ code to generate."""
-        if self.stype_values:
+        if self.feature_extended_structs:
```

Comment:
is `need_feature_generation` still needed if everything has moved endFile?

Reply:
As i mentioned before, it might not be, but until we evaluate it more thoroughly, I would just leave it.